editor/src/clj/editor/sprite.clj

```diff
@@ -592,13 +592,24 @@
 
   (input copied-nodes g/Any :array :cascade-delete)
 
-  (output aabb AABB (g/fnk [size]
+  (output aabb AABB (g/fnk [size ^:try scene-infos]
```

Comment:
is it ok to have such connection?

Reply:
Yes! It's like a try-catch, but within the graph.